### ISSUE TITLE
Missing option to  add members to a mail enabled security group through the new EAC

### ISSUE THREAD

**Issue by michelangelobottura** [MicrosoftDocs/OfficeDocs-Exchange#2662]
The new EAC does not allow to edit group owner and members: the procedure as described does not work.
 Clicking on a mail enabled security group does not open the group properties page. 

**[In the list of groups, click the mail-enabled security group that you want to view or change.
On the group's properties page, click one of the following sections to view or change properties.
 When you're finished, click Save.](https://docs.microsoft.com/en-us/exchange/recipients-in-exchange-online/manage-mail-enabled-security-groups#use-the-new-eac-to-change-mail-enabled-security-group-properties)**

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f0dd3fb4-76a1-4983-dc7a-f5cd7fb35507
* Version Independent ID: 469b3ec9-f9b1-ff8a-baaa-712114808755
* Content: [Manage mail-enabled security groups](https://docs.microsoft.com/en-us/exchange/recipients-in-exchange-online/manage-mail-enabled-security-groups)
* Content Source: [Exchange/ExchangeOnline/recipients-in-exchange-online/manage-mail-enabled-security-groups.md](https://github.com/MicrosoftDocs/OfficeDocs-Exchange/blob/public/Exchange/ExchangeOnline/recipients-in-exchange-online/manage-mail-enabled-security-groups.md)
* Service: **exchange-online**
* Technology: ****
* GitHub Login: @JoanneHendrickson
* Microsoft Alias: **jhendr**

**Comment by dariomws** [comment]
Hi @michelangelobottura, thank you for your feedback.

The steps in the article seem to be correct:
![image](https://user-images.githubusercontent.com/33589238/124141601-d7278c00-da89-11eb-9c83-e0e4e84a67f6.png)

**Comment by dariomws** [comment]
@scanum could you kindly close it? Thanks!

**Comment by michelangelobottura** [comment]
@dariomws  you looked in the wrong place: the procedure I described is related to _mail enabled security groups_ while your screenshot clearly shows you are checking it against _Microsoft 365_ groups  
![2021-08-25 10_38_26-124142072-32597e80-da8a-11eb-8310-5ead9ee057e4 png (2736×1697)](https://user-images.githubusercontent.com/50800928/130759495-1c4f64ca-4362-4837-891d-0fe2995e21a4.png).
However, I checked some days later and the new Exchange Center properly allowed members management. The documentation now reflects what can be done via the interface.
